Volume/Views/ContentView.swift

```diff
@@ -10,38 +10,110 @@ import SwiftUI
 
 struct ContentView: View {
     
-    @State private var selectedTab = 0
+    @State private var selectedTab: Tab = .home
     
     init() {
-        UITabBar.appearance().backgroundColor = UIColor(white: 250/255, alpha: 0.9)
-        UITabBar.appearance().clipsToBounds = true  // removes top border
+        let grayColor = UIColor(Color._gray)
+        UINavigationBar.appearance().backgroundColor = grayColor
+        UINavigationBar.appearance().shadowImage = UIImage()
+        UITabBar.appearance().backgroundColor = grayColor
+        UITabBar.appearance().clipsToBounds = true
         UITabBar.appearance().unselectedItemTintColor = UIColor(Color._lightGray)
     }
     
     var body: some View {
         TabView(selection: $selectedTab) {
-            Text("Home Feed")
-                .tabItem {
-                    Image("volume")
+            NavigationView {
+                HStack(alignment: .center) {
```

Comment:
Can we remove all the placeholder code since this will just be replaced with `HomeList()` right after this PR is integrated with the others